canonical-web-and-design/jaas-dashboard
# Update dependencies and lock fuzzy matches.

## Done

Update dependencies and lock fuzzy matches.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- The app should work as expected
- The demo link should work.
